php-81/Dockerfile

```diff
@@ -310,47 +107,7 @@ RUN make install
 
 
 ###############################################################################
-# Postgres
-# https://github.com/postgres/postgres/releases
-# Needs:
-#   - OpenSSL
-# Needed by:
-#   - php
-ENV VERSION_POSTGRES=15.4
```

Comment:
Native version: 15.0

Reply:
Could you cut us an issue at https://github.com/amazonlinux/amazon-linux-2023/issues detailing what from the client side you need updated?